src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/DataCollectors/SubmissionDataLockResultCollector.cs

```diff
@@ -35,7 +35,7 @@ private static SubmissionDataLockResultEntity[] ReadDataLockResultsFromDeds()
                             + " CollectionPeriodYear,"
```

Comment:
Nothing specifically wrong with what has been done in this file, but there is an inline SQL statement that "could" be converted to a stored proc. I realise its just for the acceptance tests but if this is a query used elsewhere in the application, we could benefit.